homeassistant/components/ialarm_xr/const.py

```diff
@@ -0,0 +1,18 @@
+"""Constants for the iAlarmXR integration."""
+from pyialarmxr import IAlarmXR
+
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_DISARMED,
+    STATE_ALARM_TRIGGERED,
+)
+
+DOMAIN = "ialarm_xr"
+
+IALARMXR_TO_HASS = {
```

Comment:
This is an alarm control panel platform detail that should only be part of that platform.